### PR TITLE
Feature/set default picked locale

### PR DESCRIPTION
**Pull request by akashpatl**

### CODE REVIEW

File: member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.java [hunk 1]
```diff
@@ -458,13 +458,8 @@ public void on(UpdateAcceptLanguageCommand cmd) {
...
   public void on(UpdatePickedLocaleCommand cmd) {
-
-    if (!Objects.equals(member.getPickedLocale(), cmd.getPickedLocale())) {
-      log.info("Updating picked locale for member {}, new locale: {}", cmd.getMemberId(),
-        cmd.getPickedLocale());
-
-      apply(new PickedLocaleUpdatedEvent(cmd.getMemberId(), cmd.getPickedLocale()));
-    }
+    log.info("Updating picked locale for member {}, new locale: {}", cmd.getMemberId(), cmd.getPickedLocale());
+    apply(new PickedLocaleUpdatedEvent(cmd.getMemberId(), cmd.getPickedLocale()));
```
Comment (vonElfvin) [inline member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.java:462]: Why remove the "sanity check", no point in sending the "PickedLocaleUpdatedEvent" when in reality it wasn't I'd say

File: member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.java [hunk 2]
```diff
@@ -458,13 +458,8 @@ public void on(UpdateAcceptLanguageCommand cmd) {
 
   @CommandHandler
   public void on(UpdatePickedLocaleCommand cmd) {
-
-    if (!Objects.equals(member.getPickedLocale(), cmd.getPickedLocale())) {
```
Comment (meletios) [inline member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.java:462]: we need this eller? We dont need to change the locale if its the same

**Comment by meletios** [comment]
Do we need to default it in the db also?

**Comment by johantjellden** [comment]
@FredrikAreschoug We can close this right?

**Comment by FredrikAreschoug** [comment]
yes lets just do it!

**Pull request closed without merging**